lib/checks.js

```diff
@@ -2,12 +2,14 @@
  * Copyright (c) 2012 Mathieu Turcotte
  * Licensed under the MIT license.
  */
+'use strict';
```

Comment:
So really this is about making the code run in strict mode. I'm Okay with it, but it should be reflected in the Pull request description.